src/main.cpp

```diff
@@ -59,195 +62,39 @@ namespace fs = boost::filesystem;
 #endif
 // clang-format on
 
-namespace
+namespace irods::audit
```

Comment:
I think I like the direction this is headed (that is, making the plugin operations namespaced), but if we want to expand this pattern to other plugins, maybe we would want to establish a more general naming convention? Maybe something like:
```c++
namespace irods::plugin::rule_engine::audit_amqp
```
Thoughts? Too much?

Reply:
I'm fine with that